Trials.com/react-frontend/src/components/submit-run.jsx

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Moment from "moment";
+import Star from "react-star-rating-component";
+import { useFormik, Formik } from "formik";
+import InputMask from "react-input-mask";
+import Slider from "@material-ui/core/Slider";
+import ReactStars from "react-rating-stars-component";
+
+const SubmitRun = () => {
+  const formik = useFormik({
+    initialValues: {
+      trackName: "",
+      creator: "",
+      faults: "",
+      time: "",
+      length: "",
+      faultSponginess: "",
+      ninjaLevel: "",
+      rating: "",
+    },
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  const setRating = (ratingValue) => {
+    formik.setFieldValue("rating", ratingValue);
+  };
+
+  return (
+    <Formik>
+      <form onSubmit={formik.handleSubmit}>
+        <label>Track Name</label>
```

Comment:
make sure empty can't be entered as well, i think i was able to pass it through with just blanks.

Reply:
Probably will reserve these minor changes to the overall form for others/me in the future so I can get more done to round out development.

Reply:
for sure, fine by me too